coadd1d_files/gemini_gnirs_32_sb_sxd.coadd1d

```diff
@@ -6,6 +6,6 @@
 coadd1d read
   Science/spec1d_cN20170331S0216-pisco_GNIRS_20170331T085412.181.fits OBJ0652-DET01
   Science/spec1d_cN20170331S0217-pisco_GNIRS_20170331T085933.097.fits OBJ0205-DET01
-  Science/spec1d_cN20170331S0220-pisco_GNIRS_20170331T091538.731.fits OBJ0658-DET01
-  Science/spec1d_cN20170331S0221-pisco_GNIRS_20170331T092059.597.fits OBJ0212-DET01
+  Science/spec1d_cN20170331S0220-pisco_GNIRS_20170331T091538.731.fits OBJ0659-DET01
```

Comment:
Can't we fix this annoyance for once and for all?

Reply:
I thought this was what pypeit_collate1d was for?